tests/test.py

```diff
@@ -33,6 +35,20 @@ def test_file_api(self):
         res = self.app.get("/v1/files/123?replica=aws")
         self.assertEqual(res.status_code, requests.codes.found)
 
+        res = self.app.put(
+            '/v1/files/123',
+            data=json.dumps(
+                dict(source_url="s3://foobar",
+                     bundle_uuid=str(uuid.uuid4()),
+                     creator_uid=4321,
+                     content_type="text/html",
+                     )),
```

Comment:
Also, Flask just merged support for easy JSON payload access in test fixtures (https://github.com/pallets/flask/pull/2358). It's not released yet, but it will make this boilerplate easier in the future.